src/Uno.UWP/UI/Composition/Compositor.Skia.cs

```diff
@@ -54,14 +56,13 @@ internal void Render(SKSurface surface, SKImageInfo info)
 				}
 			}
 
-			sw.Stop();
-
+			// sw.Stop();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6287442)